src/docfx/build/xref/XrefResolver.cs

```diff
@@ -212,6 +216,19 @@ private void ValidateInternalXrefProperties()
             }
         }
 
+        private void ValidateUIDGlobalUnique()
+        {
+            foreach (var (uid, source) in _internalXrefMapAndGlobalUID.Value.globalUIDs)
+            {
+                _externalXrefMap.Value.TryGetValue(uid, out var spec);
```

Comment:
Check the return value of `TryGetValue`
```suggestion
                if (_externalXrefMap.Value.TryGetValue(uid, out var _))
```

Reply:
OK.